db/migrate/20230105114342_update_materialized_patient_summaries_to_version_3.rb

```diff
@@ -0,0 +1,8 @@
+class UpdateMaterializedPatientSummariesToVersion3 < ActiveRecord::Migration[6.1]
```

Comment:
@govindkrjoshi @qptr A PR with newer migration timestamp was merged recently: https://github.com/simpledotorg/simple-server/pull/4689.

Please update the timestamp of this migration to be above `20230123125608_update_questionnaire_responses.rb`. 

Reason being, db:rollback only rolls back migration with latest timestamp.